src/gui/dosbox_staging_splash.c

```diff
@@ -22,13 +22,13 @@
       } else { __l *= 2; memcpy (__ip, __rd, __l); \
                __ip += __l; __rd += __l; } } \
   } } while (0)
-static const struct {
+static constexpr struct {
   unsigned int 	 width;
   unsigned int 	 height;
   unsigned int 	 bytes_per_pixel; /* 2:RGB16, 3:RGB, 4:RGBA */ 
-  unsigned char	 rle_pixel_data[40593 + 1];
-} gimp_image = {
-  640, 400, 3,
+  unsigned char	 rle_pixel_data[39136 + 1];
+} splash_image = {
+  640, 480, 3,
```

Comment:
Would it make sense to rendering the SVG on the fly using the single-header NanoSVG:
- https://github.com/memononen/nanosvg
- https://www.gamedev.net/forums/topic/652219-resolution-independent-graphics-with-svg-in-sdl/5125198/

Entirely cosmetic; but when you've got a logo this good, it might as well be rendered to the max :-)

Also, thinking about binary size, NanoSVG might compile down to something roughly the same size as carrying around these embedded graphics anyway.. and the benefit will be beautiful logos on latest 4k panels downt to old 1024x768 displays.

Reply:
Great suggestion! It crossed my mind long time ago, but I looked at SDL_image back then and decided to leave it be. Haven't heard about nanosvg before - definitely worth considering.

Integrating SVG support can be also seen as a prerequisite for developing better GUI - I was thinking about replacing mapper GUI with proper, nice settings screen. Most DOSBox options can be switched during runtime (DOSBox-X uses this a lot).